pkg/component/controller/kubeletconfig_test.go

```diff
@@ -92,12 +92,12 @@ func Test_KubeletConfig(t *testing.T) {
 			require.NoError(t, yaml.Unmarshal([]byte(manifestYamls[3]), &profileYYY))
 
 			// manually apple the same changes to default config and check that there is no diff
-			defaultProfileKubeletConfig := getDefaultProfile(dnsAddr, false)
+			defaultProfileKubeletConfig := getDefaultProfile(dnsAddr, false, "cluster.local")
```

Comment:
As the domain is now passed as arg, maybe also add a check that it really gets properly in place.